common/types/test_utils.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import (
+	"math"
+	"math/big"
+	mrand "math/rand"
+)
+
+func RandomID() ID {
+	id := mrand.Int63n(math.MaxInt32) + 10000
```

Comment:
```suggestion
	id := rand.Int63n(math.MaxInt32) + 10000
```
Also, why do we need the extra 10000 padding?